bin/www/codePush.js

```diff
@@ -26,8 +26,28 @@ var CodePush = (function () {
     CodePush.prototype.notifyApplicationReady = function (notifySucceeded, notifyFailed) {
         cordova.exec(notifySucceeded, notifyFailed, "CodePush", "updateSuccess", []);
     };
+    CodePush.prototype.restartApplication = function (installSuccess, errorCallback) {
```

Comment:
I'd prefer `restartApp`, since "app" is commonly used for mobile apps, whereas "application" is more for desktop apps (the actual definitions of these aren't quite as simple). This is what we went with in the React plugin, so if you feel strongly about it, let's chat :)